grpc/send_incoming_call_notification.go

```diff
@@ -0,0 +1,69 @@
+package grpc
+
+import (
+	"bytes"
+	"context"
+	"encoding/binary"
+	"log"
+	"os"
+
+	pb "github.com/PretendoNetwork/grpc-go/friends"
+	nex "github.com/PretendoNetwork/nex-go"
+	friends_wiiu "github.com/PretendoNetwork/nex-protocols-go/friends/wiiu"
+	nintendo_notifications "github.com/PretendoNetwork/nex-protocols-go/nintendo-notifications"
+	"github.com/PretendoNetwork/wiiu-chat-secure/globals"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+	"google.golang.org/grpc/metadata"
+)
+
+func SendIncomingCallNotification(caller uint32, target uint32) {
+	// Connect to Friends gRPC.
+	conn, err := grpc.Dial(os.Getenv("FRIENDS_GRPC_ADDRESS"), grpc.WithTransportCredentials(insecure.NewCredentials()))
+	if err != nil {
+		log.Fatalf("Connection to Friends gRPC failed! : %v", err)
+	}
+
+	defer conn.Close()
```

Comment:
Creating a new connection every single time isn't very efficient. I would suggest moving the client somewhere globally accessible, and only establishing the connection once at startup